### DISCUSSION TITLE
Show speed on small screen

### DISCUSSION THREAD

**Discussion by hornetster** [osmandapp/OsmAnd#19093] [Maps Related]
Have a small screen I'm using for Android Auto (Ottocast C5) with osmand+ Pro and want to show the speed. Have config'd screen to show current speed and speed limit on RHS, but these don't show.
How do I get the speed to show?

Also, am using Pro, and thought this was supposed to do Android Auto properly, but it just seems to often do the 'copout', of saying something like "turn left and follow route" . 
Shouldn't it be a little more detailed...?

Thanks.

**Comment by dmpr0** [comment]
Hi! We are currently working on a separate speedometer widget that will be available on Android Auto. All other widgets are not available on Android Auto.
